src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -71,6 +72,8 @@ public void CDSWheelToggle(boolean reverse) {
     if (reverse) {
       CDSWheelControllerOne.getSparkMax().set(-Constants.CDSWheelControllerSpeed);
       CDSWheelControllerDirection.setString("Reverse");
+      ShuffleboardTab OperatorView = Shuffleboard.getTab("CDS Speed");
```

Comment:
Should the other events go into a specific tab as well? Perhaps they could all be on OperatorView

Reply:
All of the "OperatorView" tabs should be the same tab.  
i.e:

ShuffleboardTab OperatorView = Shuffleboard.getTab("Operator View")